egs/wsj/s5/steps/nnet3/components.py

```diff
@@ -83,20 +83,51 @@ def AddOutputNode(config_lines, input, label_delay=None):
 def AddFinalLayer(config_lines, input, output_dim, ng_affine_options = "", label_delay=None, include_softmax = "true"):
```

Comment:
The LSTM specific code in components.py seems to be a lot. Could you please move the LSTM specific Add* to a separate file egs/wsj/s5/steps/nnet3/components_lstm.py.